cmd/trade.go

```diff
@@ -188,3 +193,18 @@ func deleteAllOffersAndExit(botConfig trader.BotConfig, client *horizon.Client,
 		log.Fatal("...nothing to delete, exiting")
 	}
 }
+
+func setLogFile() {
```

Comment:
we should pass in the filename var to this function

the filename `prefix` should be taken in as an optional input from the command line. If nothing is specified then it should default to std.out, i.e. it should not call the `setLogFile()` function in that case.

before calling the `setLogFile` function you can append the `timeString` (`t`) to create the `filename` passed into the function.